ts/src/cryptocom.ts

```diff
@@ -1798,68 +1800,59 @@ export default class cryptocom extends Exchange {
+        //     }
         //
-        // {
-        //     "side": "SELL",
-        //     "instrument_name": "ETH_CRO",
-        //     "fee": 0.014,
-        //     "trade_id": "367107655537806900",
-        //     "create_time": 1588777459755,
-        //     "traded_price": 7,
-        //     "traded_quantity": 1,
-        //     "fee_currency": "CRO",
-        //     "order_id": "367107623521528450"
-        // }
         const timestamp = this.safeInteger2 (trade, 't', 'create_time');
         const marketId = this.safeString2 (trade, 'i', 'instrument_name');
         market = this.safeMarket (marketId, market, '_');
-        const symbol = market['symbol'];
-        const price = this.safeString2 (trade, 'p', 'traded_price');
-        const amount = this.safeString2 (trade, 'q', 'traded_quantity');
-        let side = this.safeString2 (trade, 's', 'side');
-        if (side !== undefined) {
-            side = side.toLowerCase ();
-        }
-        const id = this.safeString2 (trade, 'd', 'trade_id');
-        const takerOrMaker = this.safeStringLower2 (trade, 'liquidity_indicator', 'taker_side');
-        const order = this.safeString (trade, 'order_id');
-        let fee = undefined;
-        let feeCost = this.safeString2 (trade, 'fee', 'fees');
-        if (feeCost !== undefined) {
-            const contract = this.safeValue (market, 'contract', false);
-            if (contract) {
-                feeCost = Precise.stringNeg (feeCost);
-            }
-            let feeCurrency = undefined;
-            if (market['spot']) {
-                feeCurrency = this.safeString (trade, 'fee_currency');
-            } else if (market['linear']) {
-                feeCurrency = market['quote'];
-            }
-            fee = {
-                'currency': feeCurrency,
-                'cost': feeCost,
-            };
-        }
+        const feeCurrency = this.safeString (trade, 'fee_instrument_name');
         return this.safeTrade ({
             'info': trade,
-            'id': id,
+            'id': this.safeString2 (trade, 'd', 'trade_id'),
             'timestamp': timestamp,
             'datetime': this.iso8601 (timestamp),
-            'symbol': symbol,
-            'side': side,
-            'price': price,
-            'amount': amount,
+            'symbol': market['symbol'],
+            'order': this.safeString (trade, 'order_id'),
+            'side': this.safeStringLower2 (trade, 's', 'side'),
+            'takerOrMaker': this.safeStringLower (trade, 'taker_side'),
+            'price': this.safeNumber2 (trade, 'p', 'traded_price'),
+            'amount': this.safeNumber2 (trade, 'q', 'traded_quantity'),
             'cost': undefined,
-            'order': order,
-            'takerOrMaker': takerOrMaker,
             'type': undefined,
-            'fee': fee,
+            'fee': {
+                'currency': this.safeCurrencyCode (feeCurrency),
+                'cost': this.safeNumber (trade, 'fees'),
```

Comment:
@Dan-krm we have to invert the sign here
![image](https://github.com/ccxt/ccxt/assets/43336371/8eba4b03-da3b-48ee-8965-ea8489eba09e)